pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -165,6 +165,24 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
 
+	// versions store the ceph version of all the ceph daemons and overall cluster version
+	versions, err := cephclient.GetAllCephDaemonVersions(c.context, c.clusterInfo)
+	if err != nil {
+		logger.Errorf("failed to get ceph daemons versions. %v", err)
+	} else {
+		// Update status with Ceph versions
+		cephCluster.Status.CephStatus.Versions = &cephv1.CephDaemonsVersions{
```

Comment:
Re-using the cephv1 type will also simplify this line.

Reply:
how it will simplify this line?